redisgraph/execution_plan.py

```diff
@@ -0,0 +1,61 @@
+class Operation:
+    def __init__(self, name, args=None):
+        self.name = name
+        self.args = args
+        self.children = []
+
+    def append_child(self, child):
+        self.children.append(child)
```

Comment:
verify that `child` is a valid Operation object, also `child` mustn't be `self` `self != child`

Reply:
done